dahu/core/app/scripts/layouts/workspace.js

```diff
@@ -6,10 +6,15 @@ define([
     'underscore',
     'handlebars',
     'backbone.marionette',
+    // views
     'views/workspace/screen',
     'views/workspace/actions',
+    'views/workspace/note',
+    // template
     'text!templates/layouts/workspace.html'
-], function(_, Handlebars, Marionette, screenView, actionsView, workspaceTemplate){
+], function(_, Handlebars, Marionette, 
```

Comment:
Please respect the same format as in 'dahuapp.js', namely:

define([
    ... deps paths ..
], function(
    ... module ...
) {
  ... code ...
});